star-rx72n-firmware/Dockerfile

```
@@ -1,14 +1,158 @@
+# Build a modern RX GNU toolchain with C23 support.
+
+FROM ubuntu:22.04 AS toolchain
+
+ARG BINUTILS_VERSION=2.41
+ARG GCC_VERSION=13.2.0
+ARG NEWLIB_VERSION=4.3.0.20230120
+ARG MAKE_JOBS=1
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)